pkg/controller/masteruserrecord/masteruserrecord_controller_test.go

```diff
@@ -384,7 +384,7 @@ func TestModifyUserAccounts(t *testing.T) {
 	userAccount2 := uatest.NewUserAccountFromMur(mur)
 	userAccount3 := uatest.NewUserAccountFromMur(mur)
 
-	murtest.ModifyUaInMur(mur, test.MemberClusterName, murtest.NsLimit("advanced"), murtest.TierName("admin"), murtest.Namespace("ide", "54321"))
+	murtest.ModifyUaInMur(mur, test.MemberClusterName, murtest.NsLimit("advanced"), murtest.TierName("admin"), murtest.Namespace("basic-dev-123abc", "admin-ide-54321"))
```

Comment:
do you know that you are replacing different tier and different namespace type?

Reply:
fixed as part of: 000a2b5594241a958671b11904043853eb3fbb47